infrastructure/resource_group_level/modules/webapps/trade-finance-manager-api.bicep

```diff
@@ -39,9 +39,7 @@ param secureConnectionStrings object = {
   CORS_ORIGIN: 'test-value'
   UKEF_TFM_API_SYSTEM_KEY: 'test-value'
   UKEF_TFM_API_REPORTS_KEY: 'test-value'
-  // TODO:FN-429 Note that this has a value like https://tfs-dev-tfm-fd.azurefd.net 
-  // while in the CLI it is injected as a secret, we can probably calculate it from the Front Door component.
-  TFM_URI: 'test-value'
+  TFM_UI_URL: 'test-value'
```

Comment:
I observed that `TFM_URI` is probably able to be derived. Can this be too, or does it need to come from a secret?

Reply:
can be derived as well

Reply:
Oh wait; I see why it was set via a secret - there's a circular dependency.
`TfmApi` -> `TfmUi` -> `ApplicationGratewayTfm` -> `TfmFrontdoor` -> `TfmApi`.

I'll make an explicit card to think about that.